core/templates/components/skill-selector/merge-skill-modal.component.ts

```diff
@@ -28,12 +28,12 @@ import { SkillsCategorizedByTopics } from 'pages/topics-and-skills-dashboard-pag
   templateUrl: './merge-skill-modal.component.html'
 })
 export class MergeSkillModalComponent extends ConfirmOrCancelModal {
-  categorizedSkills: SkillsCategorizedByTopics;
-  skill: AugmentedSkillSummary;
-  skillSummaries: AugmentedSkillSummary[];
-  untriagedSkillSummaries: SkillSummary[];
+  categorizedSkills!: SkillsCategorizedByTopics;
+  skill!: AugmentedSkillSummary;
+  skillSummaries!: AugmentedSkillSummary[];
+  untriagedSkillSummaries!: SkillSummary[];
```

Comment:
Please add a comment explaining why `!` are added. You can find this comment in many existing files.

Reply:
Done!